recipe/83ef352c821f6f5092d4132e3ecd5d5eb6b4e9e8.patch

```diff
@@ -0,0 +1,34 @@
+From 83ef352c821f6f5092d4132e3ecd5d5eb6b4e9e8 Mon Sep 17 00:00:00 2001
+From: Andrei Capastru <andreicap@users.noreply.github.com>
+Date: Tue, 4 Aug 2020 11:00:56 +0200
+Subject: [PATCH] Replace df.ix with df.loc in tools.py
+
+`pandas.DataFrame.ix` is deprecated and does not work in newer versions.
```

Comment:
Can you clarify what newer means here? In which version were these functions removed?